provider/resources.go

```diff
@@ -157,39 +158,41 @@ func Provider() tfbridge.ProviderInfo {
 			"github_user_ssh_key":                  {Tok: makeResource(mainMod, "UserSshKey")},
 		},
 		DataSources: map[string]*tfbridge.DataSourceInfo{
-			"github_actions_dependabot_organization_secrets": {
-				Tok: makeDataSource(mainMod, "getActionsDependabotOrganizationSecrets"),
-			},
-			"github_actions_dependabot_secrets": {
-				Tok: makeDataSource(mainMod, "getActionsDependabotSecrets"),
-			},
```

Comment:
Were these renamed? That would be a breaking schema change, wouldn't it?

Reply:
update: seems like these were mistakenly added - I don't see any corresponding changes in the SDKs so this looks safe 👍